src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json

```diff
@@ -0,0 +1,62 @@
+{
+  "_guid_" : "GRANT_CASE_ROLE_ASSIGNMENT",
```

Comment:
for all files using the /am/role-assignments  endpoint lets create and extend a base file like F-201_GrantRoleAssignment.td.json and put the common lines of code here like the api details and request headers, then we can remove this common lines from all the files extending them